sql/core/src/test/scala/org/apache/spark/sql/SparkSessionExtensionSuite.scala

```diff
@@ -151,14 +241,63 @@ case class MyParser(spark: SparkSession, delegate: ParserInterface) extends Pars
 object MyExtensions {
 
   val myFunction = (FunctionIdentifier("myFunction"),
-    new ExpressionInfo("noClass", "myDb", "myFunction", "usage", "extended usage" ),
+    new ExpressionInfo("noClass", "myDb", "myFunction", "usage", "extended usage"),
     (myArgs: Seq[Expression]) => Literal(5, IntegerType))
+
```

Comment:
I would remove this newline. Looks unrelated.

Reply:
Done